### ISSUE TITLE
HTTPS website

### ISSUE THREAD

**Issue by jayvdb** [Psychtoolbox-3/psychtoolbox-3.github.com#8]
Hi, I noticed that https://psychtoolbox.org/ shows "Warning: Potential Security Risk Ahead" in Firefox because it has a wrong certificate for that domain.

https://help.github.com/en/github/working-with-github-pages/securing-your-github-pages-site-with-https may be of help to resolve this.

**Comment by kleinerm** [comment]
Yes, that's a problem. So far we have tried, but not yet managed to switch to https, because apparently of some issues with our DNS provider. I hope we'll eventually manage, but i'm not
the person responsible for these tasks or DNS maintenance.

**Comment by krispenney** [comment]
:wave: I noticed that this was still a problem, were there any updates on this?

**Comment by skjerns** [comment]
any fix on this?

**Comment by kleinerm** [comment]
Nope.

**Comment by kleinerm** [comment]
Fixed now.
